.github/workflows/jlpkgbutler-codeformat-pr-workflow.yml

```diff
@@ -1,10 +1,6 @@
 name: Code Formatting
 
 on:
-  push:
-    branches:
-      - main
-      - master
```

Comment:
Those files should not be manually edited, they are managed by PkgButlerEngine.jl.

Reply:
I undid this change.